### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by ftrasi** [johngodley/redirection#668]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 5.6.31-4+wpengine10
Browser: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: El nonce de la cookie no es válido
Action: redirection/v1/404 GET
Code: 403 
Raw: {"code":"rest_cookie_invalid_nonce","message":"El nonce de la cookie no es v\u00e1lido","data":{"status":403}}
```

**Comment by johngodley** [comment]
Can you try the 3.1 beta here:

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
